### PR TITLE
160-moreIntegrationTests

### PR DESCRIPTION
**Pull request by TobiasNx**
A lot more integration tests

### CODE REVIEW

**Comment by blackwinter** [comment]
Test names should start with lowercase (`If` -> `if`).

**Review by blackwinter — CHANGES_REQUESTED** [review]
Typo: `Refrence` -> `Reference`

Expected files must be formatted correctly; I've given up marking them individually...

File: metafix/src/test/resources/org/metafacture/metafix/integration/record/fromJson/toJson/set_arrayNestedSimple/test.fix [hunk 1]
```diff
@@ -0,0 +1,2 @@
+set_array("object.test[]")
+set_array("object_2.test_2[]", "dog", "cat", "bird")
```
Comment (blackwinter) [inline metafix/src/test/resources/org/metafacture/metafix/integration/record/fromJson/toJson/set_arrayNestedSimple/test.fix:2]: Should probably be two tests.

**Review by blackwinter — CHANGES_REQUESTED** [review]
Typo still present: `Refrence` -> `Reference`

File: metafix/src/test/resources/org/metafacture/metafix/integration/record/fromJson/toJson/set_arrayNestedSimple/test.fix (hunk 1 above)
Comment (TobiasNx) [reply metafix/src/test/resources/org/metafacture/metafix/integration/record/fromJson/toJson/set_arrayNestedSimple/test.fix]: I have split the `set_array` tests. Found an error and added some more variations of the tests.

**Review by blackwinter — APPROVED** [review]

**Pull request merged**